components/vf-utility-classes/vf-u-width.scss

```diff
@@ -0,0 +1,37 @@
+// Creates responsive sizes like:
+// .vf-width__10 { width: 10%; }
+// .vf-width__10--sm { width: 10%; }
+
+@for $i from 1 through 20 {
+  .vf-u-width__#{$i*5} { width: 5%*$i; }
```

Comment:
As you're doing `#{$i*5]` can we do `$i * 5%`? whitespace is your friend ;)

Reply:
Makes sense. Pushing a change.